reference/glossary.md

```diff
@@ -0,0 +1,47 @@
+# Glossary
+
+This glossary defines the terminology used to describe the various concepts related to a Webots simulation.
+
+**Actuator**: An *actuator* is a *node* representing a real robotics actuator such as a motor, a LED, a radio emitter, etc. *Actuators* may receive commands from a *controller*. Their behavior will affect the outcome of a simulation. Some *actuators* are also *sensors* if they make sensory measurement, such as a motor than can provide position or force feedback.

```

Comment:
You are right. I will remove the position feedback from this sentence. But although it is an exception, I believe we should keep this comment about force feedback.